src/blob/index/bptree/tests.rs

```diff
@@ -65,12 +65,29 @@ impl Into<usize> for KeyType {
     }
 }
 
+fn get_test_dir(dir_name: &str) -> PathBuf {
+    let result = std::env::temp_dir()
+        .join("pearl_bptree_test")
+        .join((std::time::UNIX_EPOCH.elapsed().unwrap().as_secs() % 1_563_100_000).to_string())
+        .join(dir_name);
+
+    std::fs::create_dir_all(&result).expect("Directory created");
+
+    return result;
+}
+
+fn clean(index: BPTreeFileIndex<KeyType>, path: impl AsRef<Path>) -> Result<()> {
```

Comment:
Since every occurence of `clean` is accompanied by `expect` it's probably better to remove return value from this function and move `expect` inside.